backend/helpers/mailer.js

```diff
@@ -0,0 +1,14 @@
+const nodemailer = require("nodemailer");
+const config = require("../config");
+const gettemplates = require("./emailTemplates");
+
+module.exports = sendEmail;
+
+async function sendEmail({ to, from = config.smtpOptions.auth.user, details, tempname}) {
+    var template = gettemplates(details);
```

Comment:
It will still unnecessarily parse all the templates with your details which is not required.
What we are looking for is that the emailTemplates will be a json file from which specific template can be extracted and sent to a function which will parse and replace the required variables.